Dockerfile.deploy

```diff
@@ -56,16 +56,18 @@ RUN localedef -i en_US -f UTF-8 en_US.UTF-8
 ENV LANG en_US.UTF-8
 ENV LC_ALL en_US.UTF-8
 
+# Install uwsgi statsd exporter to collect metrics from uwsgi when deployed
+WORKDIR /usr/lib/uwsgi/plugins
+RUN uwsgi --build-plugin https://github.com/Datadog/uwsgi-dogstatsd && \
+    rm -rf uwsgi-dogstatsd
```

Comment:
Why remove the file?

Reply:
This is removing the git repo that uwsgi downloads and builds. Done in the same RUN command to save space in the layer.